server/db/recreate-schema.sql

```diff
@@ -44,7 +37,7 @@ CREATE TABLE assessments(
 CREATE TABLE questions(
    question_id INT GENERATED ALWAYS AS IDENTITY,
    assessment_id INT NOT NULL,
-   question_text VARCHAR(5000) NOT NULL,
+   question_text VARCHAR(255) NOT NULL,
```

Comment:
question_text will be more than 250 characters, so it's better to have it large than small